### DISCUSSION TITLE
Env Variable

### DISCUSSION THREAD

**Discussion by BigShrimpGuy** [jontze/cadency-rs#145] [Q&A] [answered]
I am brand new to coding, how do I change the Environment Viarable? I have the image on Docker Desktop and I am not sure if I have to edit it through visual studio or somehow through Docker.

**Comment by jontze** [comment, marked as answer]
Hey @BigShrimpGuy,
if you run Cadency through the Docker container, you have to set the environment variables in the container. You should be able to configure this somewhere within Docker Desktop. Unfortunately, I only use the Docker Engine via CLI and can't point you to the correct location in the desktop version.

However, if I remember correctly, the `docker` CLI tool should be available in your (Windows?) terminal if you installed `Docker Desktop`. Therefore, you could alternatively start the container and configure the environment variables as explained in the README [here](https://github.com/jontze/cadency-rs?tab=readme-ov-file#example-docker-setup).

I hope this helps 😄
